### DISCUSSION TITLE
why call it preact, what's the 'p' meaning ?

### DISCUSSION THREAD

**Discussion by yolio2003** [preactjs/preact#3936] [Q&A] [answered]
**just want to know how to understand the name, thx!**

**Comment by dmorenogogoleva** [comment]
perfect react?

**Comment by rschristian** [comment]
purple react

**Comment by developit** [comment, marked as answer]
It doesn't really mean anything. Folks have suggested "petit React" (French for small react), which is hilarious.

**Comment by marvinhagemeister** [comment]
poetic react

**Comment by Shyam-Chen** [comment]
- `performant` + `react` = `preact`
- `performant` + `npm` = `pnpm`
- ...
